src/corefx/System.Globalization.Native/configure.cmake

```diff
@@ -8,6 +8,17 @@ CHECK_C_SOURCE_COMPILES("
     int main(void) { enum UDateFormatSymbolType e = UDAT_STANDALONE_SHORTER_WEEKDAYS; }
 " HAVE_UDAT_STANDALONE_SHORTER_WEEKDAYS)
 
+CHECK_C_SOURCE_COMPILES("
+#include <stdatomic.h>
+int main(void)
+{
+    // check for https://bugs.llvm.org/show_bug.cgi?id=37457
+    int tmp;
+    atomic_store_explicit((_Atomic(int) *)&tmp, 0, memory_order_relaxed);
```

Comment:
Is the cast from `int*` to `_Atomic(int)` portable? It would be better to make `_Atomic(int) tmp` instead of the cast.